### DISCUSSION TITLE
[Camera Support]: Wsse authorized time check failed

### DISCUSSION THREAD

**Discussion by nwish** [blakeblackshear/frigate#14310] [Camera Support]
### Describe the problem you are having

![IMG_4554](https://github.com/user-attachments/assets/ddce8f72-28af-431e-b63c-69f271d96638)

Anyone experienced this error before?  ONVIF connection to the camera had been working in the past for quite some time and I honestly didn’t notice when it stopped.  It’s an Amcrest Wifi PTZ.  Haven’t changed anything other than continuing to pull the dev branch.  I didn’t want to report a bug just yet because it may be a camera thing.  After some searching, I found some other reports of issues with the camera being set to a different time zone but I have ensured that everything is the same and synced properly.  

This log entry appears when I navigate to the “den” camera, which is the only one of 6 that has an ONVIF connection configured in Frigate.  

### Version

0.15.0-acccc6f

### What browser(s) are you using?

Various, Chrome / Safari 

### Frigate config file

```yaml
.
```

### Relevant Frigate log output

```shell
.
```

### Relevant go2rtc log output

```shell
.
```

### FFprobe output from your camera

```shell
.
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Mixed

### Camera make and model

Amcrest PTZ

### Screenshots of the Frigate UI's System metrics pages

.

**Comment by nwish** [comment]
I suppose I should add that this is preventing Frigate from making the connection and showing the PTZ controls.

**Reply by hawkeye217** [reply]
Onvif uses timestamps to prevent replay attacks. Frigate and your camera exchange timestamps, and that error indicates that there's a mismatch between them.

This error usually occurs when the time on your server is significantly different from the time on the camera, there are timezone inconsistencies between the server and the camera, or one of the devices has an incorrect date set.

**Reply by nwish** [reply]
Huh.  Okay, I’ll play around some more.  Both devices are within a few seconds of each other and they both sync with the time server on the router.

**Reply by hawkeye217** [reply]
You can try setting `ignore_time_mismatch: True` under the `onvif` section of that camera in your config file. 

Perhaps it's a daylight savings time setting in the Amcrest firmware? Frigate's onvif code hasn't changed, did you recently upgrade the firmware on that camera?
